### ISSUE TITLE
3.0.0-beta2 throws npe

### ISSUE THREAD

**Issue by farukcankaya** [intercom/intercom-android#199]
## Version info
  - Intercom for Android version: 3.0.0-beta2
  - Android OS version: 4.4.2
  - Android Device:GT-I9190

## Expected behavior
crash-free

## Actual behavior
throws npe

## Logs
``````
Fatal Exception: java.lang.NullPointerException
       at io.intercom.android.sdk.conversation.ConversationFragment$3.onImageReady(ConversationFragment.java:282)
       at io.intercom.android.sdk.imageloader.LongTermImageLoader$Fetcher$1.onResourceReady(LongTermImageLoader.java:87)
       at io.intercom.android.sdk.imageloader.LongTermImageLoader$Fetcher$1.onResourceReady(LongTermImageLoader.java:80)
       at io.intercom.com.bumptech.glide.request.GenericRequest.onResourceReady(GenericRequest.java:11525)
       at io.intercom.com.bumptech.glide.load.engine.EngineJob.handleResultOnMainThread(EngineJob.java:1158)
       at io.intercom.com.bumptech.glide.load.engine.EngineJob$MainThreadCallback.handleMessage(EngineJob.java:202)
       at android.os.Handler.dispatchMessage(Handler.java:98)
       at android.os.Looper.loop(Looper.java:146)
       at android.app.ActivityThread.main(ActivityThread.java:5593)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:515)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1283)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1099)
       at dalvik.system.NativeStart.main(NativeStart.java)
``````

**Comment by ronocod** [comment]
Hi @farukcankaya,

Thanks for reporting this, we've fixed this bug and the fix will be in the next version. I'll reply here when it's released.

**Comment by harkin** [comment]
This one is fixed now, update to `3.0.0-beta3` :)

**Comment by farukcankaya** [comment]
Thanks for quick action :) @harkin
